src/Rules/Library/ChromiumComponentsShouldUseWebScanner.cs

```diff
@@ -27,7 +26,7 @@ public override bool PassesTest(IA11yElement e)
 
         protected override Condition CreateCondition()
         {
-            return Chrome & (Document | AnyAncestor(Document));
+            return Chrome & Document;
```

Comment:
This is the change that reduces the scope of `ChromiumComponentsShouldUseWebScanner` from _all_ Chromium elements to just Chromium _Documents_